src/settings-ui/Settings.UI/SettingsXAML/Views/CmdNotFoundPage.xaml

```diff
@@ -21,16 +21,29 @@
         <controls:SettingsPageControl.ModuleContent>
             <StackPanel ChildrenTransitions="{StaticResource SettingsCardsAnimations}" Orientation="Vertical">
                 <InfoBar
-                    x:Uid="GPO_SettingIsManaged"
+                    x:Uid="GPO_CommandNotFound_ForceEnabled"
                     IsClosable="False"
-                    IsOpen="{x:Bind ViewModel.IsEnabledGpoConfigured, Mode=OneWay}"
-                    IsTabStop="{x:Bind ViewModel.IsEnabledGpoConfigured, Mode=OneWay}"
-                    Severity="Informational" />
+                    IsOpen="{x:Bind ViewModel.IsModuleGpoEnabled, Mode=OneWay}"
+                    IsTabStop="{x:Bind ViewModel.IsModuleGpoEnabled, Mode=OneWay}"
+                    Severity="Informational">
+                    <InfoBar.IconSource>
+                        <FontIconSource FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE72E;" />
+                    </InfoBar.IconSource>
+                </InfoBar>
+                <InfoBar
+                    x:Uid="GPO_CommandNotFound_ForceDisabled"
+                    IsClosable="False"
+                    IsOpen="{x:Bind ViewModel.IsModuleGpoDisabled, Mode=OneWay}"
+                    IsTabStop="{x:Bind ViewModel.IsModuleGpoDisabled, Mode=OneWay}"
+                    Severity="Informational">
+                    <InfoBar.IconSource>
+                        <FontIconSource FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE72E;" />
+                    </InfoBar.IconSource>
+                </InfoBar>
```

Comment:
Adjust to have signle bar and switch text in ViewModel.

Reply:
After thinking about possible changes I think  the current implementation is the best one.